### ISSUE TITLE
hacs not working with the latest HASS

### ISSUE THREAD

**Issue by heapxor** [hacs/integration#2226]
### System Health details

          

Version | core-2021.9.7
-- | --
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.7
Operating System Family | Linux
Operating System Version | 5.10.17-v8
CPU Architecture | aarch64
Timezone | Europe/Prague

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have enabled debug logging for my installation.
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the backend (integration part) of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).

### Describe the issue

Hi,
i installed HASS on rpi4 ; went step by step to install HACS but i got error

2021-09-27 00:10:56 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration frigate which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-27 00:10:56 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: system_health/info
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: system_health/info
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:11:05 ERROR (MainThread) [custom_components.hacs] GitHub API ratelimited - 1008 remaining
2021-09-27 00:11:05 ERROR (MainThread) [custom_components.hacs] HACS is disabled - HacsDisabledReason.RATE_LIMIT
2021-09-27 00:11:05 WARNING (MainThread) [homeassistant.config_entries] Config entry '' for hacs integration not ready yet: None; Retrying in background

not sure if its HACS issue or HASS or frigate issue?

running Home Assistant 2021.9.7
HACS Integration version: | 1.15.2

if i go to HACS -> information>
HACS is disabled (Ratelimited)
GitHub API calls are ratelimited, this will clear in less than 1 hour.

more logs >
**http://paste.debian.net/1213378/
http://paste.debian.net/1213380/**

thanks

### Reproduction steps

https://hacs.xyz/docs/configuration/basic

### Debug logs

```text
2021-09-27 00:10:56 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration frigate which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-27 00:10:56 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: system_health/info
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: system_health/info
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:10:59 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [548460526320] Received invalid command: energy/get_prefs
2021-09-27 00:11:05 ERROR (MainThread) [custom_components.hacs] GitHub API ratelimited - 1008 remaining
2021-09-27 00:11:05 ERROR (MainThread) [custom_components.hacs] HACS is disabled - HacsDisabledReason.RATE_LIMIT
2021-09-27 00:11:05 WARNING (MainThread) [homeassistant.config_entries] Config entry '' for hacs integration not ready yet: None; Retrying in background
```

**Comment by ludeeus** [comment]
It is, try again later

**Comment by heapxor** [comment]
@ludeeus it is?
its not working even 12 hours passed.

What do I have to try?

**Comment by ludeeus** [comment]
a restart

**Comment by heapxor** [comment]
@ludeeus okay, i restarted and now it looks like it works! thanks!

**Comment by heapxor** [comment]
@ludeeus 
when i go to step > 
HACS does not "configure" the integration for you. You must go to Configuration > Integrations and add Frigate after installing via HACS.



2021-09-27 11:29:45 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration frigate which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-27 11:29:45 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-09-27 11:29:48 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [547924266240] Received invalid command: system_health/info
2021-09-27 11:32:31 ERROR (MainThread) [custom_components.frigate.api] Error fetching information from http://ccab4aaf-frigate:5000/api/stats: Cannot connect to host ccab4aaf-frigate:5000 ssl:default [Try again]
2021-09-27 11:32:42 ERROR (MainThread) [custom_components.frigate.api] Error fetching information from http://ccab4aaf-frigate:5000/api/stats: Cannot connect to host ccab4aaf-frigate:5000 ssl:default [Try again]
